mmv1/products/firebase/api.yaml

```diff
@@ -152,3 +213,9 @@ objects:
           Immutable. The globally unique, Firebase-assigned identifier of the App.
 
           This identifier should be treated as an opaque token, as the data format is not specified.
+      - !ruby/object:Api::Type::Array
```

Comment:
was this intended to be in this pull request?

Reply:
No, I mistakenly branched this off the first PR branch instead of main.